app/actions/websocket/threads.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {markTeamThreadsAsRead, processReceivedThreads, toggleFollowThread, updateThreadRead} from '@actions/local/thread';
+import DatabaseManager from '@database/manager';
+
+export async function handleThreadUpdatedEvent(serverUrl: string, msg: WebSocketMessage): Promise<void> {
+    try {
+        const thread = JSON.parse(msg.data.thread) as Thread;
+        const teamId = msg.broadcast.team_id;
+
+        // Mark it as following and visible in global threads
+        thread.is_following = true;
+        thread.loaded_in_global_threads = true;
```

Comment:
this will now change with #6100

Reply:
Removed it from here .. I will address it in the Global Threads PR as we have to make the changes according to the changes.